src/main/webapp/script.js

```diff
@@ -16,13 +16,33 @@ $('#myModal').on('shown.bs.modal', function () {
     $('#myInput').trigger('focus')
 });
 
-function initializeModal(pathId, itemId, itemName) {
-    console.log("initializing", arguments)
+function initializeModal(itemId, itemName) {
+    console.log("initializing", arguments);
 
     $("#feedback-title").text(itemName);
+    $("#feedback-id").text(itemId);
     const form = $("#feedback");
 }
 
 function submitFeedback() {
-  // TODO
-}
+    console.log("Sending the submission form info.");
+    const itemId = document.getElementById("feedback-id").innerText;
+    const pathId = document.getElementById("path-id").innerText;
+    const options = document.getElementById("ratingValues");
+    const rating = options.options[options.selectedIndex].value;
+
+    const headers = new Headers();
+    headers.append("rating", rating);
+    //If done is always going to be true, is
+    //it really necessary to be sent?
+    headers.append("done", "true");
```

Comment:
Headers should represent request options, common ones include 'Content-Type', 'Accept-Encoding', or 'Authorization'. The actual data should not be passed in the headers

Reply:
`FormData` should go in the `body` attribute:

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Body